pkgs/applications/misc/sweethome3d/default.nix

```diff
@@ -80,6 +84,10 @@ let
         --prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS:${gtk3.out}/share:${gsettings-desktop-schemas}/share:$out/share:$GSETTINGS_SCHEMAS_PATH" \
         --add-flags "-Dsun.java2d.opengl=true -jar $out/share/java/${module}-${version}.jar -cp $out/share/java/Furniture.jar:$out/share/java/Textures.jar:$out/share/java/Help.jar -d${toString stdenv.hostPlatform.parsed.cpu.bits}"
 
+
+      # remember the store paths found inside the .jar libraries
+      cp .jar-paths $out
```

Comment:
Can we find a better place for the file? Otherwise it ends up in ``/``.

Reply:
Would you be good with `/nix-support/depends`? I see it on #46141 and Bazel already uses it for this same purpose: https://github.com/NixOS/nixpkgs/blob/af56903adb67202dbca2a8b31d3b162b0ee71cd6/pkgs/development/tools/build-managers/bazel/bazel_0_26/default.nix#L379-L386

Reply:
That sounds better to me if it works.